<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="openImgButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Open image</string>
    </property>
   </widget>
   <widget class="QPushButton" name="flipVerticalButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Flip vertically</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>30</y>
      <width>491</width>
      <height>311</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="flipHorButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Flip horizontally</string>
    </property>
   </widget>
   <widget class="QPushButton" name="flipHorVerButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>230</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Flip hor/ver</string>
    </property>
   </widget>
   <widget class="QPushButton" name="switchColorButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>300</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Switch colors (RGB &lt;-&gt; BGR)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveImgButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>370</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Save image</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
